run-tests.pl

```diff
@@ -645,8 +645,8 @@ sub _push_test
    }
 
    if( exists $params{implementation_specific} ) {
-       if ( $HS_FACTORY->implementation_name() ne $params{implementation_specific} ) {
-           return;
+       unless (grep { $_ eq $HS_FACTORY->implementation_name() } @{$params{implementation_specific}}) {
```

Comment:
looks like you either need to update *all* the `implementation_specific` declarations, or make this cope with strings as well as lists.

Reply:
Bleh, will update them all because like hell I'm modifying this line.

Reply:
hehe

could you update `DEVELOP.rst`, which mentions this, too?